api/src/Page/Shipment.php

```diff
@@ -925,6 +1000,27 @@ function _dispatch_dewar() {
                 array($dew['DEWARID'], $dewar_location)
             );
 
+            $data = $this->args;
```

Comment:
I know this is just continuing the pattern but could we lift this in to a function (I am not sure what the arguments look like but it feels like it separates off that bit of code